index/index.go

```diff
@@ -973,23 +973,23 @@ func (t *stringTuples) Less(i, j int) bool {
 }
 
 type serializedStringTuples struct {
-	l      int
-	b      []byte
-	lookup func(uint32) (string, error)
+	idsCount   int
+	idsEntries []byte
```

Comment:
As this is not exactly entries but the bytes containing them, I would ideally name them `idsBytes` (previous naming was `b`, I guess it meant bytes).

It was not totally clear why prefix is `ids` (later saw the comments). Can you add a comment saying these bytes contains the ids pointing to actual string in the lookup table.

Reply:
yep +1 on both comments.

Reply:
So @krasi-georgiev what should I do? Please don't mind me asking as I don't quite fully understand.

Reply:
```
idsBytes []byte // bytes containing the ids pointing to the string in the lookup table.
```

Reply:
Updated